third_party/java/jetty/BUILD.bazel

```diff
@@ -5,7 +5,7 @@ licenses(["notice"])
 
 package(default_visibility = ["//visibility:public"])
 
-JETTY_VERSION = "9.4.12.v20180830"
+JETTY_VERSION = "9.4.19.v20190610"
```

Comment:
@shs96c I updated the version numbers everywhere I could see them in the BUILD.bazel file, and the build seemed to work fine. What are you seeing that I'm missing? It should all be here in the PR, so let me know what changes need to be made. Thanks!

Reply:
Ah! This might actually work :) I didn't see this before.